models/intel/person-reidentification-retail-0248/description/person-reidentification-retail-0248.md

```diff
@@ -10,14 +10,14 @@ feature map outputs an embedding vector of 256 floats.
 
 ## Example
 
-![](./person-reidentification-retail-0107.jpg)
+![](./person-reidentification-retail-0248.jpg)
 
 ## Specification
 
 | Metric                            | Value                                     |
 |-----------------------------------|-------------------------------------------|
-| Internal-reID-test rank@1 accuracy| 91.7%                                     |
-| Internal-reID-test mAP            | 63.4%                                     |
+| Market-1501 rank@1 accuracy       |  %                                    |
```

Comment:
is expected accuracy known for the model?

Reply:
The accuracy is known